src/shared/utils/number-to-US-locale.test.ts

```diff
@@ -19,75 +19,75 @@
  */
 
 /* global describe, test, expect */
-import numberToUSLocale from './number-to-US-locale'
+import numberToUSLocale from './number-to-US-locale';
```

Comment:
I think it'd be nice to leave of the semi colons for typescript as well to avoid these kinds of diffs every time we port something to ts. Thoughts?

Reply:
I agree, semicolons should match between js and ts